Tools/ardupilotwaf/boards.py

```diff
@@ -549,6 +549,10 @@ def configure_env(self, cfg, env):
             APJ_BOARD_ID = 100,
             HAL_NO_GCS = 1,
             HAL_LOGGING_ENABLED = 0,
```

Comment:
we should enable logging for sitl_gps I think.

Reply:
I didn't want to fiddle with your working setup :-)

One issue I can see with enabling it is that both the main firmware you'll probably be running and the SITL GPS want to log into the same directory!

I think we should consider that problem at a future date?

Reply:
The reason of adding is that it automatically will put your logging changes as part of CI run. But yeah, I don't think this should hold this PR.